### DISCUSSION TITLE
Our rating has slipped to 1784:

### DISCUSSION THREAD

**Discussion by BonzoDogDooDah** [lightninglabs/lightning-terminal#759] [Q&A]
Hi, Our node rating was going lower to 439 then a couple of days later it shout right up to 1784? Now instead of a Good + it says Stable? We have all of our channels set to Auto Fees so were wondering why this has happened? Please let us know. Thank you :) BonzoDogDooDah

**Comment by bitromortac** [comment]
Your node barely has the minimal number of required good peers, so if a channel goes inactive (because of a peer being down perhaps), it can happen that you drop out from the list temporarily until that channel is up again (right now your node is back on the list). To be more resilient against this, you could open one or two channels to good peers.
